### PR TITLE
general translation fix

### PR DESCRIPTION
**Pull request by aBOOminacja**

### CODE REVIEW

**Comment by vincentkerdraon** [comment]
Perfect, I see you were able to create a PR.
(I just changed the base branch to point to `master`)

Another I need to ask you is: Can you configure your editor to avoid line breaks.

For example it was:
```
msgstr "Twoje konto Klei zostało pomyślnie utworzone! Wypełnij formularz, aby go aktywować i mieć dostęp do dodatkowych funkcji. Możesz również zapisać się do naszego newslettera, aby otrzymywać informacje o swojej ulubionej grze."
```
And your editor is changing it to
```
msgstr ""
"Twoje konto Klei zostało pomyślnie utworzone! Wypełnij formularz, aby go "
"aktywować i mieć dostęp do dodatkowych funkcji. Możesz również zapisać się do "
"naszego newslettera, aby otrzymywać informacje o swojej ulubionej grze."
```

This is making it really hard to see what are the changes. (I am talking about the "Files changed" tab in the PR).

For example I can quickly see the changes you made to 
```
msgid "Betas"
msgstr "Bety"
```

And also, our system does not support multiline format.

I don't know much about your .po editor but I think there must be an option to configure that.

I am sorry to ask you that many technical changes, but we need those steps to be able to deal with new translations.

**Comment by aBOOminacja** [comment]
i will try to fix it but in fact i dont know what happened

File: pl-PL/strings.po [hunk 1]
```diff
@@ -3,19 +3,19 @@ msgstr ""
...
+"PO-Revision-Date: 2021-07-27 21:37+0200\n"
+"Last-Translator: aBOOminacja\n"
 "Language-Team: Polish (https://accounts.klei.com/community-translations)\n"
 "Language: pl_PL\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<12"
-"X-Crowdin-Project: klei-accounts\n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<12X-Crowdin-Project: klei-accounts;\n"
```
Comment (vincentkerdraon) [inline pl-PL/strings.po:13]: Here a line break is missing.

**Review by vincentkerdraon — CHANGES_REQUESTED** [review]
The format of the file is now good, I see only a little mistake in the header.
After you fix it, I will merge that (send the changes on this branch to the branch master)
+ deploy the changes.

Nearly there!

File: pl-PL/strings.po (hunk 1 above)
Comment (aBOOminacja) [reply pl-PL/strings.po]: done?

**Comment by vincentkerdraon** [comment]
Great job!
This is deployed with your latest changes.

```
 _   _                 _                      
| | | |               | |                     
| |_| |__   __ _ _ __ | | ___   _  ___  _   _ 
| __| '_ \ / _` | '_ \| |/ / | | |/ _ \| | | |
| |_| | | | (_| | | | |   <| |_| | (_) | |_| |
 \__|_| |_|\__,_|_| |_|_|\_\\__, |\___/ \__,_|
                             __/ |            
                            |___/    
```

**Pull request merged**